app/src/main/java/com/github/h3lp3rs/h3lp/AwaitHelpActivity.kt

```diff
@@ -248,6 +247,12 @@ class AwaitHelpActivity : AppCompatActivity() {
         startActivity(intent)
     }
 
+    /**
+     * Launches the RecentMessagesActivity with the given emergency key to display the messaging
+     * screen
```

Comment:
```suggestion
     * screen.
```

Reply:
Definitely not nitpicking 🐦

Reply:
Obviously not :) (also the dot doesn't follow our comments convention.. so I guess this is counter-nitpicking).